procedures/observing_tools/Sky Almanac.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>It does more than that :)</p><p>It also plots the elevation and airmass for a series of designated targets!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>